src/main.rs

```diff
@@ -36,7 +36,7 @@ Default options are `build` and `test`
 ");
 
 #[derive(Copy)]
-struct Config {
+pub struct Config {
```

Comment:
Actually I just tried this and the compiler complains about a `private type in exported type signature` if we don't specify `pub`.